node/builder.go

```diff
@@ -274,6 +275,7 @@ func Online() Option {
 
 			Override(new(*paychmgr.Store), paychmgr.NewStore),
 			Override(new(*paychmgr.Manager), paychmgr.NewManager),
+			Override(HandlePaymentChannelManagerKey, paychmgr.HandleManager),
```

Comment:
On startup we need to restart tracking of payment channel add funds requests that have already been sent to chain